template/src/features/counter/counter.module.css

```diff
@@ -0,0 +1,31 @@
+.container {
```

Comment:
In terms of naming convention, I've always preferred to make my CSS module files exactly match the name of the corresponding component files.  In this case, that'd be capitalizing it to `Counter.module.css`.